### PR TITLE
TRITON-2404 Good-bye node v4

### PR DESCRIPTION
**Pull request by bahamat**
To test this I ran the node-triton test suite.

The test suite did have some failures, but the reason for those failures are not related to workflow:

| failure | reason |
| --- | --- |
| vm&nbsp;rollback | rollback and destroy were sent at the same time, causing destroy to succeed and rollback to fail because the vm no longer exists. workflow did the right thing |
| nic&nbsp;changes | `add nic`, `remove nic`, `add nic` jobs were all sent simultaneously. The first add triggers a reboot, causing the second add to fail. The remove succeeded because it's just updating the vm json. workflow did the right thing |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
   "name": "workflow",
-  "description": "SmartDataCenter Workflow API and Runners",
-  "version": "1.4.1",
-  "author": "Joyent (joyent.com)",
+  "description": "Triton Workflow API and Runners",
+  "version": "1.4.2",
+  "author": "MNX Cloud (mnx.ia)",
```
Comment (danmcd) [inline package.json]: typo:  `mnx.io` not `mnx.ia`.

**Review by danmcd — COMMENTED** [review]
One typo.

File: docs/index.md [hunk 2]
```diff
@@ -20,7 +21,10 @@ with specific modules, configurations and the required tools to
 build and setup workflow zones either into SDC Headnode or Compute Nodes.
 
 Please, note this documentation is not about the general purpose `wf` and backend modules.
-Documentation for these is publicly available at [Terminology and system description](http://kusor.github.io/node-workflow/ "This includes how wf-runners work") and the [Workflow REST API](http://kusor.github.io/node-workflow/workflowapi.html).
+Documentation for these is publicly available at
+[Terminology and system description](http://kusor.github.io/node-workflow/ "This includes how wf-runners work")
```
Comment (travispaul) [inline docs/index.md]: These links no longer work

Comment (danmcd) [reply docs/index.md]: Perhaps https://github.com/TritonDataCenter/node-workflow ?

Comment (bahamat) [reply docs/index.md]: Yeah, will fix.

File: package.json (hunk 1 above)
Comment (bahamat) [reply package.json]: No idea how I did that :-)

**Review by danmcd — APPROVED** [review]

**Review by travispaul — APPROVED** [review]

**Pull request merged**